src/sentry/rules/actions/base.py

```diff
@@ -74,3 +75,21 @@ def get_integration(self):
 
     def get_form_instance(self):
         return self.form_cls(self.data, integrations=self.get_integrations())
+
+
+class TicketEventAction(IntegrationEventAction):
+    """Shared ticket actions"""
+
+    def build_description(self, event, installation, newline=False, pipe=False):
+        rule_url = u"/organizations/{}/alerts/rules/{}/{}/".format(
+            self.project.organization.slug, self.project.slug, self.rule.id
+        )
+        footer = u"This ticket was automatically created by Sentry via "
+        if pipe:
```

Comment:
idk if this is a great var name but I couldn't think of a better one

Reply:
@ceorourke I think the better approach would be to have a method called `generate_footer` that we can call here and let Azure and Jira have different implementations of that method. It's cleaner than having to pass these sorts of args when we call the function.